Snakefile

```diff
@@ -1318,6 +1318,8 @@ rule build_cop_profiles:
 
 
 rule prepare_heat_data:
+    params:
+        sector_options=config["sector"],
```

Comment:
It would be probably safer to define a more narrow sub-set of `config["sector"]` parameters to avoid triggering unneeded re-runs of a workflow due to a change of some parameters which are actually not relevant for `prepare_heat_data`. What do you think?